### DISCUSSION TITLE
DreamBerd now supports rich text

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#616] [Announcements]
In this week's weekly update, we have a very exciting update for you. Let's just say, we're... **rich** to the brim with excitement (ha ha ha ha).

## DreamBerd now supports rich text

Have you guessed it yet? Yes, you guessed it! DreamBerd now supports rich text.

<pre>
const const <b>name</b> = "Lu"!
const const <i>name</i> = "Luke"!

print(<b>name</b>)! // Lu
print(<i>name</i>)! // Luke
</pre>

Rich text can be helpful when making your website. Use it to add links!

<pre>
&lt;p>Click <a href="https://dreamberd.computer">here</a>&lt;/p>
</pre>

## Bounty update

Here's an update on our affluent and highly successful [bounty program](https://github.com/TodePond/DreamBerd/tree/main/docs/investment/bounty). GitHub user [mgrkan](https://github.com/mgrkan) successfully completed the `£99 to a charity of your choice if you comment below saying "I miss when dreamberd was just about the programming and not any of this woke rubbish"` bounty, and so £99 has been donated to [UNICEF](https://www.unicef.org/) as requested by them. UNICEF is providing [life-saving support to children in Gaza](https://www.unicef.org/emergencies/children-gaza-need-lifesaving-support).

See you next week.

<img width="1087" alt="Screenshot 2024-05-13 at 17 03 30" src="https://github.com/TodePond/DreamBerd/assets/15892272/b3e55982-666e-4075-a739-655e96343b97">

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v%C2%A3.%C2%A3%C2%A3'>DreamBerd now supports rich text</a>.</em>

**Comment by Golem642** [comment]
I miss when dreamberd was just about the programming and not any of this woke rubbish

**Reply by TodePond** [reply]
nice try but it's already claimed :)

**Reply by Golem642** [reply]
I miss when bounties were just about the donations and not any of this claimed rubbish

**Comment by dbarenholz** [comment]
Arguably, if we get this discussion to 99+ likes, then todepond can donate 99 _ponds_ to a charity of their own choice. I quote `£99 to a charity of your choice if you write a post primarily about dreamberd on any microblogging platform that gets over 99 likes.` from the [bounty program](<https://github.com/TodePond/DreamBerd/tree/main/docs/investment/bounty>).

**Comment by mybearworld** [comment]
<pre><code>const const data = ["John", "Doe", "π"]!

print(<table><thead><th align=center>"First Name"</th><th align=center>"Last Name"</th><th align=center>"Age"</th></thead><tbody><tr><td align=center>data[-1]</td><td align=center>data[0]</td><td align=center>data[1]</td></tr></tbody></table>)!</code></pre>

**Reply by Rudxain** [reply]
JS `console.table` has been dead silent since this `drop`ped 🔥🔥💯

**Comment by felistachio** [comment]
Can you link to the comment/post where the "I miss when dreamberd was just about the programming and not any of this woke rubbish" bounty was completed?

**Reply by Toby222** [reply]
https://github.com/TodePond/DreamBerd/discussions/597#discussioncomment-8719595

**Comment by kjloveless** [comment]
@TodePond Has this bounty been paid?
> £99 to a charity of your choice if you add gifs to bluesky (not dreamberd related) (I just want it to happen).

https://github.com/bluesky-social/social-app/pull/3605

May be worth reaching out to Hailey about it?

**Comment by MaddyGuthridge** [comment]
Can you make variable names support markdown syntax as well?

```dreamberd3
const const **name** = "Maddy"!
const const _name_ = "Maddy"!
const const #name = "Maddy"!  // I only have one name I think
```

All text in error messages displays using the rendered markdown, for example:

```dreamberd
#name + _name_
```

`TypeError`:

# name

(type `string`) cannot be added to _name_ (type `char[]`) due to incompatible types

**Reply by mybearworld** [reply]
You could probably have an editor extension do that to output this code.

<pre>
const const <b>name</b> = "Maddy"!
const const <i>name</i> = "Maddy"!
const const <h1>name</h1> = "Maddy"!
</pre>
